<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>upd_user</class>
 <widget class="QWidget" name="upd_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(89, 108, 134);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Изменение Пользователя</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Фамилия</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Номер телефона</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="inputMask">
      <string>+7(000)000-00-00</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
